src/ERC7540Vault.sol

```diff
@@ -293,8 +295,13 @@ contract ERC7540Vault is Auth, IERC7540 {
     }
 
     // --- Helpers ---
-    function exchangeRateLastUpdated() external view returns (uint64) {
-        return manager.exchangeRateLastUpdated(address(this));
+    /// @notice Price of 1 unit of share, quoted in the decimals of the asset
+    function pricePerShare() external view returns (uint256) {
```

Comment:
This methods enables looking up the price, without also needing to look up the decimals of the tranche tokens (thus also saving gas).